chef/cookbooks/ceph/attributes/radosgw.rb

```diff
@@ -30,14 +30,14 @@
 default["ceph"]["radosgw"]["nss_directory"] = "/var/lib/ceph/nss"
 
 case node["platform"]
-  when "ubuntu"
-    default["ceph"]["radosgw"]["service_name"]  = "radosgw-all-starter"
-  when "debian"
-    default["ceph"]["radosgw"]["service_name"]  = "radosgw"
-  when "suse"
-    default["ceph"]["radosgw"]["service_name"]  = "ceph-radosgw@#{node["hostname"]}"
-  else
-    default["ceph"]["radosgw"]["service_name"]  = "ceph-radosgw"
+when "ubuntu"
+  default["ceph"]["radosgw"]["service_name"] = "radosgw-all-starter"
+when "debian"
+  default["ceph"]["radosgw"]["service_name"] = "radosgw"
+when "suse", "opensuse"
+  default["ceph"]["radosgw"]["service_name"] = "ceph-radosgw@#{node["hostname"]}"
+else
+  default["ceph"]["radosgw"]["service_name"] = "ceph-radosgw"
```

Comment:
Somehow, I feel we should move more towards `platform_family` (to avoid bad surprises for new distros). What about something like:
```ruby
case node["platform_family"]
  when "debian"
    if node["platform"] == "ubuntu"
      default["ceph"]["radosgw"]["service_name"]  = "radosgw-all-starter"
    else
      default["ceph"]["radosgw"]["service_name"]  = "radosgw"
    end
  when "suse"
    default["ceph"]["radosgw"]["service_name"]  = "ceph-radosgw@#{node["hostname"]}"
  else
    default["ceph"]["radosgw"]["service_name"]  = "ceph-radosgw"
end
```